manof/image.py

```diff
@@ -1,6 +1,8 @@
 import os
+import sys
 import pipes
 import inspect
+import utils
```

Comment:
import order

Reply:
you mean newline before `import utils` ? 👍 
Alphabetical import order in python is not a thing in our linters today

Reply:
newline, yes